ui/server/src/main/resources/web/static/assets/states/deploy-running-test-animated.svg

```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.cls-1{fill:none;}.cls-2{fill:#666;}.cls-3{fill:#ffd550;}.cls-4{fill:#2983cc;}.cls-5{fill:#fff;}</style></defs><title>deploy-running-test-light</title><g id="Layer_2" data-name="Layer 2"><g id="ikonsy"><path class="cls-1" d="M17.34,9.58a2.06,2.06,0,1,0-2.92,0A2.06,2.06,0,0,0,17.34,9.58Z"/><path class="cls-2" d="M18.13,13l1.51-1.38a12.25,12.25,0,0,0,3.86-7.5l.49-3A1,1,0,0,0,22.85,0l-3,.49a12.25,12.25,0,0,0-7.5,3.86L9,8,4.45,7.76a.68.68,0,0,0-.61.31L1.54,11.5a.69.69,0,0,0,.29,1L5.07,14,10,18.93l1.47,3.24a.69.69,0,0,0,1,.29l1.41-.93A5.49,5.49,0,0,1,18.13,13ZM14.42,6.66a2.06,2.06,0,1,1,0,2.92A2.06,2.06,0,0,1,14.42,6.66Z"/><path class="cls-2" d="M4.45,15.73a.27.27,0,0,0-.38,0l-.6.59A12.25,12.25,0,0,0,0,24a12.25,12.25,0,0,0,7.69-3.47l.59-.6a.27.27,0,0,0,0-.38Z"/>

```

Comment:
let's leave it for next PR? Or use them where there are needed - e.g. in specific deployments?